api/server/router/volume/volume.go

```diff
@@ -30,6 +30,7 @@ func (r *volumeRouter) initRoutes() {
 		// POST
 		router.NewPostRoute("/volumes/create", r.postVolumesCreate),
 		router.NewPostRoute("/volumes/prune", r.postVolumesPrune, router.WithCancel),
+		router.NewPostRoute("/volumes/rename", r.postVolumeRename),
```

Comment:
I think /volumes/{name:.*}/rename could make the url consistent, since we have already had a API
```
router.NewPostRoute("/containers/{name:.*}/rename", r.postContainerRename),
```
WDYT?